diff --git a/jax/_src/api.py b/jax/_src/api.py
index d91fa92..e45d6a1 100644
--- a/jax/_src/api.py
+++ b/jax/_src/api.py
@@ -692,7 +692,7 @@ def fwd_and_bwd(
   argnums = _ensure_index(argnums)
 
   def fwd(*args, **kwargs):
-    f_partial, dyn_args = argnums_partial2(fun, argnums, args, {})
+    f_partial, dyn_args = argnums_partial2(fun, argnums, args, kwargs)
     return vjp(f_partial, *dyn_args, has_aux=has_aux)
   def bwd(f_vjp, outgrad):
     g = f_vjp(outgrad)
